2023-06-15 22:14:28 - WARNING - aliyundrive.py - 820 - Token is about to expire, try to auto refresh.
2023-06-15 22:31:47 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-15 22:31:48 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-15 22:48:40 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2023/06/03/14/31/22/108683773_p0.jpg
2023-06-15 22:52:09 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2022/12/23/00/00/19/103816502_p0.jpg
2023-06-16 10:16:28 - WARNING - aliyundrive.py - 820 - Token is about to expire, try to auto refresh.
2023-06-16 10:27:14 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-16 10:27:15 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-16 10:27:45 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-16 10:27:47 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-16 10:28:17 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-16 10:28:20 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-16 10:28:51 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-16 10:28:55 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2023-06-16 10:28:55 - ERROR - aliyundrive.py - 1034 - Failed to get create info of file tmp/98726093/98726093.json.txt.
2023-06-16 10:29:27 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-16 10:29:28 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-16 10:49:30 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2023/03/11/17/33/54/106114864_p0.jpg
2023-06-16 11:09:34 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))': /ajax/illust/106058359
2023-06-16 11:38:44 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2022/04/29/00/00/12/97947478_p0.png
2023-06-16 11:46:28 - WARNING - pixivdrive.py - 127 - Failed to add salt to page tmp/107454934/107454934_ugoira0.gif, upload original page.
2023-06-16 11:46:28 - WARNING - pixivdrive.py - 155 - Failed to make mirage for page tmp/107454934/107454934_ugoira0.gif.
2023-06-16 11:56:29 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2023/02/18/14/19/11/105430019_p0.png
2023-06-16 12:15:30 - WARNING - aliyundrive.py - 820 - Token is about to expire, try to auto refresh.
2023-06-16 22:14:16 - WARNING - aliyundrive.py - 820 - Token is about to expire, try to auto refresh.
2023-06-16 22:20:56 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2023/01/21/00/00/06/104675991_p0.png
2023-06-16 22:34:37 - ERROR - base.py - 94 - https://api.aliyundrive.com/v2/file/complete:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-16 22:34:38 - WARNING - base.py - 25 - Retry func _post_file_complete 1 time.
2023-06-16 22:40:02 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2023/06/12/20/59/20/108958356_p0.png
2023-06-16 23:05:19 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2023/05/29/23/58/45/108557574_p0.png
2023-06-16 23:16:29 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-16 23:16:30 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-16 23:34:29 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2023/02/26/00/51/52/105711534_p0.jpg
2023-06-16 23:37:35 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2023/05/29/13/01/45/108541730_p0.jpg
2023-06-16 23:40:50 - WARNING - pixivdrive.py - 127 - Failed to add salt to page tmp/107454934/107454934_ugoira0.gif, upload original page.
2023-06-16 23:40:50 - WARNING - pixivdrive.py - 155 - Failed to make mirage for page tmp/107454934/107454934_ugoira0.gif.
2023-06-17 00:03:33 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2023/04/16/10/35/06/107221625_p0.jpg
2023-06-17 10:15:19 - WARNING - aliyundrive.py - 820 - Token is about to expire, try to auto refresh.
2023-06-17 10:50:43 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-17 10:50:44 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-17 10:51:15 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-17 10:51:17 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-17 10:51:47 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-17 10:51:50 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-17 10:52:20 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-17 10:52:24 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2023-06-17 10:52:24 - ERROR - aliyundrive.py - 1034 - Failed to get create info of file tmp/102006529/102006529.json.txt.
2023-06-17 10:52:56 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-17 10:52:57 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-17 11:01:33 - WARNING - pixivdrive.py - 127 - Failed to add salt to page tmp/108150135/108150135_ugoira0.gif, upload original page.
2023-06-17 11:01:34 - WARNING - pixivdrive.py - 155 - Failed to make mirage for page tmp/108150135/108150135_ugoira0.gif.
2023-06-17 11:21:47 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-17 11:21:48 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-17 11:22:18 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-17 11:22:20 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-17 11:22:50 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-17 11:22:53 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-17 11:23:24 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-17 11:23:28 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2023-06-17 11:23:28 - ERROR - aliyundrive.py - 1034 - Failed to get create info of file tmp/94970370/94970370_p0.png.
2023-06-17 11:23:58 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-17 11:23:59 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-17 11:24:29 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-17 11:24:31 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-17 11:41:49 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))': /ajax/illust/104568886/pages
2023-06-17 11:46:51 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-17 11:46:52 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-17 11:47:22 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-17 11:47:24 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-17 11:47:54 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-17 11:47:57 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-17 11:48:27 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-17 11:48:31 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2023-06-17 11:48:31 - ERROR - aliyundrive.py - 1034 - Failed to get create info of file tmp/103278518/103278518_p1.png.
2023-06-17 11:49:02 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-17 11:49:03 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-17 11:59:17 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-17 11:59:18 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-17 11:59:48 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-17 11:59:50 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-17 12:00:21 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-17 12:00:24 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-17 12:00:54 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-17 12:00:58 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2023-06-17 12:00:58 - ERROR - aliyundrive.py - 1034 - Failed to get create info of file tmp/106525134/106525134.json.txt.
2023-06-17 12:01:29 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-17 12:01:30 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-17 12:03:38 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-17 12:03:39 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-17 12:04:09 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-17 12:04:11 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-17 12:04:41 - ERROR - base.py - 94 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-17 12:04:44 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-17 12:06:48 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2023/06/17/10/19/24/109082075_p0.jpg
2023-06-17 12:07:03 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2022/10/23/00/31/41/102152635_p0.jpg
2023-06-17 12:14:19 - WARNING - aliyundrive.py - 820 - Token is about to expire, try to auto refresh.
2023-06-17 22:14:11 - WARNING - aliyundrive.py - 820 - Token is about to expire, try to auto refresh.
2023-06-17 22:59:38 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2022/09/22/00/04/06/101393406_p0.jpg
2023-06-17 23:04:22 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2023/05/10/21/43/01/108009307_p0.png
2023-06-18 00:13:13 - WARNING - aliyundrive.py - 820 - Token is about to expire, try to auto refresh.
2023-06-18 00:31:46 - WARNING - pixivdrive.py - 127 - Failed to add salt to page tmp/108150135/108150135_ugoira0.gif, upload original page.
2023-06-18 00:31:46 - WARNING - pixivdrive.py - 155 - Failed to make mirage for page tmp/108150135/108150135_ugoira0.gif.
2023-06-18 10:15:45 - WARNING - aliyundrive.py - 820 - Token is about to expire, try to auto refresh.
2023-06-18 10:18:34 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d216870835147055978e5dbd","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 10:18:34 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 10:18:35 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-18 10:18:36 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d216870835159766093e5dbd","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 10:18:36 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 10:18:38 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-18 10:18:38 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d216870835182226262e5dbd","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 10:18:38 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 10:18:41 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-18 10:18:41 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d216870835214706637e5dbd","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 10:18:41 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 10:18:45 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2023-06-18 10:18:45 - ERROR - aliyundrive.py - 954 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2023-06-18 10:23:43 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070ce16870838234835840e8555","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 10:23:43 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 10:23:44 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-18 10:23:44 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070ce16870838247406081e8555","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 10:23:44 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 10:23:46 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-18 10:23:47 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070ce16870838270206612e8555","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 10:23:47 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 10:23:50 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-18 10:23:50 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070ce16870838302817238e8555","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 10:23:50 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 10:23:54 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2023-06-18 10:23:54 - ERROR - aliyundrive.py - 954 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2023-06-18 10:31:21 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d316870842812528751eefdc","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 10:31:21 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 10:31:22 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-18 10:31:22 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d316870842825098931eefdc","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 10:31:22 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 10:31:24 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-18 10:31:24 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d316870842847831255eefdc","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 10:31:24 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 10:31:27 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-18 10:31:28 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d316870842880431605eefdc","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 10:31:28 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 10:31:32 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2023-06-18 10:31:32 - ERROR - aliyundrive.py - 954 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2023-06-18 10:40:42 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800316870848419997139e2791","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 10:40:42 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 10:40:43 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-18 10:40:43 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800316870848432587317e2791","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 10:40:43 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 10:40:45 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-18 10:40:45 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800316870848455177628e2791","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 10:40:45 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 10:40:48 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-18 10:40:48 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800316870848488027975e2791","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 10:40:48 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 10:40:52 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2023-06-18 10:40:52 - ERROR - aliyundrive.py - 954 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2023-06-18 10:49:24 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d516870853639094521ed90e","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 10:49:24 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 10:49:25 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-18 10:49:25 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d516870853651654642ed90e","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 10:49:25 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 10:49:27 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-18 10:49:27 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d516870853674234888ed90e","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 10:49:27 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 10:49:30 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-18 10:49:30 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d516870853706875125ed90e","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 10:49:30 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 10:49:34 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2023-06-18 10:49:34 - ERROR - aliyundrive.py - 954 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2023-06-18 10:53:24 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d416870856040431715e3ef2","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 10:53:24 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 10:53:25 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-18 10:53:25 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d416870856052971828e3ef2","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 10:53:25 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 10:53:27 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-18 10:53:27 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d416870856075542047e3ef2","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 10:53:27 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 10:53:30 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-18 10:53:30 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d416870856108412324e3ef2","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 10:53:30 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 10:53:34 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2023-06-18 10:53:34 - ERROR - aliyundrive.py - 954 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2023-06-18 10:56:05 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800716870857656792546ea0cb","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 10:56:05 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 10:56:06 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-18 10:56:07 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800716870857669642705ea0cb","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 10:56:07 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 10:56:09 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-18 10:56:09 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800716870857692452963ea0cb","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 10:56:09 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 10:56:12 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-18 10:56:12 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800716870857725263446ea0cb","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 10:56:12 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 10:56:16 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2023-06-18 10:56:16 - ERROR - aliyundrive.py - 954 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2023-06-18 11:04:31 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800816870862712568364ede19","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:04:31 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:04:32 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-18 11:04:32 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800816870862725128538ede19","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:04:32 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:04:34 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-18 11:04:34 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800816870862747638749ede19","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:04:34 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:04:37 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-18 11:04:38 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800816870862780191053ede19","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:04:38 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:04:42 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2023-06-18 11:04:42 - ERROR - aliyundrive.py - 954 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2023-06-18 11:05:55 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800216870863551833489e6a3e","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:05:55 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:05:56 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-18 11:05:56 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800216870863564393670e6a3e","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:05:56 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:05:58 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-18 11:05:58 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800216870863587004087e6a3e","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:05:58 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:06:01 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-18 11:06:02 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800216870863619614611e6a3e","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:06:02 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:06:06 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2023-06-18 11:06:06 - ERROR - aliyundrive.py - 954 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2023-06-18 11:08:26 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2022/07/13/02/18/09/99687660_p0.png
2023-06-18 11:09:57 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d216870865973015118e5dc0","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:09:57 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:09:58 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-18 11:09:58 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d216870865985595330e5dc0","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:09:58 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:10:00 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-18 11:10:00 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d216870866008185613e5dc0","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:10:00 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:10:03 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-18 11:10:04 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d216870866040785996e5dc0","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:10:04 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:10:08 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2023-06-18 11:10:08 - ERROR - aliyundrive.py - 954 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2023-06-18 11:14:40 - ERROR - base.py - 94 - https://api.aliyundrive.com/v2/file/complete:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-18 11:14:41 - WARNING - base.py - 25 - Retry func _post_file_complete 1 time.
2023-06-18 11:15:27 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2022/12/04/01/35/29/103317483_p0.png
2023-06-18 11:19:41 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800716870871810846023ea0c6","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:19:41 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:19:42 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-18 11:19:42 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800716870871823716144ea0c6","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:19:42 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:19:44 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-18 11:19:44 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800716870871846576395ea0c6","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:19:44 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:19:47 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-18 11:19:48 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800716870871879556664ea0c6","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:19:48 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:19:52 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2023-06-18 11:19:52 - ERROR - aliyundrive.py - 954 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2023-06-18 11:21:30 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0080fb16870872902435971e9839","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:21:30 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:21:31 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-18 11:21:31 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0080fb16870872915056162e9839","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:21:31 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:21:33 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-18 11:21:33 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0080fb16870872937656535e9839","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:21:33 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:21:36 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-18 11:21:37 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0080fb16870872970227091e9839","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:21:37 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:21:41 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2023-06-18 11:21:41 - ERROR - aliyundrive.py - 954 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2023-06-18 11:27:48 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d716870876681431585ed902","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:27:48 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:27:49 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-18 11:27:49 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d716870876693971782ed902","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:27:49 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:27:51 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-18 11:27:51 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d716870876716482070ed902","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:27:51 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:27:54 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-18 11:27:55 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d716870876749222553ed902","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:27:55 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:27:59 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2023-06-18 11:27:59 - ERROR - aliyundrive.py - 954 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2023-06-18 11:32:16 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d316870879362788128eefd2","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:32:16 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:32:17 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-18 11:32:17 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d316870879375328216eefd2","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:32:17 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:32:19 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-18 11:32:19 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d316870879397878401eefd2","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:32:19 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:32:22 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-18 11:32:23 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d316870879430468692eefd2","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:32:23 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:32:27 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2023-06-18 11:32:27 - ERROR - aliyundrive.py - 954 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2023-06-18 11:47:57 - ERROR - base.py - 94 - https://api.aliyundrive.com/v2/file/complete:HTTPSConnectionPool(host='api.aliyundrive.com', port=443): Read timed out. (read timeout=30)
2023-06-18 11:47:58 - WARNING - base.py - 25 - Retry func _post_file_complete 1 time.
2023-06-18 11:48:49 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800116870889292265488e8127","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:48:49 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:48:50 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-18 11:48:50 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800116870889304765639e8127","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:48:50 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:48:52 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-18 11:48:52 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800116870889327315861e8127","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:48:52 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:48:55 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-18 11:48:56 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800116870889359916211e8127","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:48:56 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:49:00 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2023-06-18 11:49:00 - ERROR - aliyundrive.py - 954 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2023-06-18 11:49:31 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800116870889709552162e8127","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:49:31 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:49:32 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-18 11:49:32 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800116870889722092258e8127","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:49:32 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:49:34 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-18 11:49:34 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800116870889744882524e8127","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:49:34 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:49:37 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-18 11:49:37 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800116870889777502942e8127","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 11:49:37 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 11:49:41 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2023-06-18 11:49:41 - ERROR - aliyundrive.py - 954 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2023-06-18 12:00:17 - WARNING - connectionpool.py - 751 - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /img-original/img/2022/12/04/19/20/37/103337166_p0.jpg
2023-06-18 12:01:59 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d716870897194468533ed8fa","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:01:59 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:02:00 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-18 12:02:00 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d716870897207228669ed8fa","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:02:00 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:02:02 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-18 12:02:03 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d716870897230078928ed8fa","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:02:03 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:02:06 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-18 12:02:06 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d716870897263131210ed8fa","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:02:06 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:02:10 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2023-06-18 12:02:10 - ERROR - aliyundrive.py - 954 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2023-06-18 12:08:55 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070cf16870901353102423e7d9e","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:08:55 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:08:56 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-18 12:08:56 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070cf16870901365652558e7d9e","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:08:56 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:08:58 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-18 12:08:58 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070cf16870901388442774e7d9e","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:08:58 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:09:01 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-18 12:09:02 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070cf16870901421063022e7d9e","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:09:02 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:09:06 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2023-06-18 12:09:06 - ERROR - aliyundrive.py - 954 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2023-06-18 12:14:41 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d316870904817057189eefe0","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:14:41 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:14:42 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-18 12:14:43 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d316870904830157289eefe0","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:14:43 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:14:45 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-18 12:14:45 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d316870904853327516eefe0","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:14:45 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:14:48 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-18 12:14:48 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d316870904886427821eefe0","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:14:48 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:14:52 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2023-06-18 12:14:52 - ERROR - aliyundrive.py - 954 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2023-06-18 12:14:52 - WARNING - aliyundrive.py - 820 - Token is about to expire, try to auto refresh.
2023-06-18 12:20:30 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d716870908302907157ed907","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:20:30 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:20:31 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-18 12:20:31 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d716870908315467308ed907","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:20:31 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:20:33 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-18 12:20:34 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d516870908345133130ed913","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:20:34 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:20:37 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-18 12:20:37 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d516870908377953386ed913","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:20:37 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:20:41 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2023-06-18 12:20:41 - ERROR - aliyundrive.py - 954 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2023-06-18 12:21:34 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d516870908945881929ed913","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:21:34 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:21:35 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-18 12:21:36 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d516870908959222015ed913","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:21:36 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:21:38 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-18 12:21:38 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d516870908981852199ed913","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:21:38 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:21:41 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-18 12:21:41 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d516870909014512503ed913","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:21:41 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:21:45 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2023-06-18 12:21:45 - ERROR - aliyundrive.py - 954 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2023-06-18 12:38:16 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800a16870918967358294efc9d","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:38:16 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:38:17 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-18 12:38:18 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800a16870918979918424efc9d","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:38:18 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:38:20 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-18 12:38:20 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800a16870919002468615efc9d","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:38:20 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:38:23 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-18 12:38:23 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800a16870919034988896efc9d","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:38:23 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:38:27 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2023-06-18 12:38:27 - ERROR - aliyundrive.py - 954 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2023-06-18 12:42:16 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d216870921363408063e5dbe","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:42:16 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:42:17 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-18 12:42:17 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d216870921376018234e5dbe","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:42:17 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:42:19 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-18 12:42:20 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d216870921398618515e5dbe","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:42:20 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:42:23 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-18 12:42:23 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a0070d216870921431268905e5dbe","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:42:23 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:42:27 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2023-06-18 12:42:27 - ERROR - aliyundrive.py - 954 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2023-06-18 12:50:38 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800816870926385375277ede0f","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:50:38 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:50:39 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-18 12:50:39 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800816870926397955388ede0f","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:50:39 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:50:41 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-18 12:50:43 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800916870926428511421ed18b","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:50:43 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:50:46 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-18 12:50:46 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800916870926461631833ed18b","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:50:46 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:50:50 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2023-06-18 12:50:50 - ERROR - aliyundrive.py - 954 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
2023-06-18 12:52:10 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800916870927304315103ed18b","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:52:10 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:52:11 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 1 time.
2023-06-18 12:52:11 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800916870927317105259ed18b","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:52:11 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:52:13 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 2 time.
2023-06-18 12:52:14 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800916870927339885501ed18b","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:52:14 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:52:17 - WARNING - base.py - 25 - Retry func _post_file_create_with_folders 3 time.
2023-06-18 12:52:17 - WARNING - base.py - 100 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:{"code":"FileNameDisable","message":"file name disable","requestId":"0a00800916870927372925799ed18b","resultCode":"FileNameDisable","display_message":"请修改名称，不能包含字符 /*?:<>\"|"}
2023-06-18 12:52:17 - ERROR - aliyundrive.py - 80 - https://api.aliyundrive.com/adrive/v2/file/createWithFolders:403:file name disable
2023-06-18 12:52:21 - ERROR - base.py - 36 - All retries failed in func _post_file_create_with_folders.
2023-06-18 12:52:21 - ERROR - aliyundrive.py - 954 - Failed to create folder pixiv/56627683/SWKL:D - 56627683.
